python/tank/errors.py

```diff
@@ -19,6 +19,12 @@ class TankError(Exception):
     """
 
 
+class TankInitError(TankError):
```

Comment:
Should we avoid creating anything `Tank*`? Maybe call this `ShotgunIntegrationsInitError` or `ToolkitInitError`?

Reply:
it's a consistency thing - all things inherited from TankError are TankSomething right now so i just followed that pattern.